platforms/native/testing-native/src/integTest/groovy/org/gradle/nativeplatform/test/xctest/AbstractSwiftXCTestComponentIntegrationTest.groovy

```diff
@@ -32,6 +32,9 @@ abstract class AbstractSwiftXCTestComponentIntegrationTest extends AbstractSwift
     def setup() {
         // TODO: Temporarily disable XCTests with Swift3 on macOS
         Assume.assumeFalse(OperatingSystem.current().isMacOsX() && toolChain.version.major == 3)
+
+        // Need XCTest available to run these tests
```

Comment:
XCTest is distributed in XCode, not the command-line tools, so skip the tests if XCode is not installed.